VS/VehicleSignals.rdf

```diff
@@ -4687,6 +4695,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;ThrottleActuator">
 		<rdfs:subClassOf rdf:resource="&sosa;Actuator"/>
+		<rdfs:label xml:lang="en">Throttle actuator</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.